blocks/url-input/index.js

```diff
@@ -61,6 +61,17 @@ class UrlInput extends Component {
 			return;
 		}
 
+		// Disable suggestions for URLs
+		if ( 0 === value.indexOf( 'http://' ) || 0 === value.indexOf( 'https://' ) ) {
```

Comment:
Do we have an `isURL()` or similar helper for all protocols?

Reply:
Maybe use a regex to catch `any://`

```js
if ( /^\w+:\/\//.test( value ) ) {
```

There's also the built-in URL module:
https://nodejs.org/docs/latest-v7.x/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost

Maybe a good place for other URL utilities:
https://github.com/WordPress/packages/blob/master/packages/url/src/index.js

Reply:
> Maybe use a regexp to catch `any://`

I think this would be more durable, if we don't want to introduce a dedicated utility.

Reply:
70dc202b5dbc369dd8799038e5839a28aaa80ece updates the check to use the regex our existing TinyMCE plugin uses.